art/attacks/fast_gradient.py

```diff
@@ -108,6 +109,16 @@ def _minimal_perturbation(self, x, y, eps_step=0.1, eps_max=1., **kwargs):
 
         return adv_x
 
+    def _get_rate(self, x_adv, y):
```

Comment:
I believe this function is not used anymore and can be removed?

Reply:
Yes, resolved with 163ed9778f63d067e1565ba15f488773c2722673